### PR TITLE
Glide version locked

### PR DESCRIPTION
**Pull request by alexkhimich**
Locked Glide versions for reproducible builds

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ before_install:
 
 install:
   - glide install
+  - cd cmd/apid ; glide install
```
Comment (theganyo) [inline .travis.yml]: Why did you add this? It's not needed for testing.

File: glide.yaml [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+  version: 5ed0fc31f7f453625df314d8e66b9791e8d13003
 - package: github.com/mattn/go-sqlite3
-  version: master
+  version: v1.2.0
 - package: github.com/gorilla/mux
-  version: master
+  version: v1.3.0
 - package: github.com/30x/goscaffold
-  version: master
+  version: cbed6858015606bec730ced4b5423392667d69c6
```
Comment (theganyo) [inline glide.yaml]: This is internal. Do we really want to pin it to a specific commit now?

Comment (alexkhimich) [inline glide.yaml]: yes,

Comment (alexkhimich) [inline glide.yaml]: i want to update on stable builds

Comment (theganyo) [inline glide.yaml]: ok for now, I suppose... but we should probably label our builds, then.
